perl-uri.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: perl-uri
-  version: 5.19
+  version: 5.20
   epoch: 0
```

Comment:
```suggestion
  epoch: 1
```

Reply:
Why this?

Reply:
you're right, I'm an idiot lost in severance